src/main/views/home.njk

```diff
@@ -1,8 +1,11 @@
+{% from "macros/csrf.njk" import csrfProtection %}
+
 <html>
   <head>
   </head>
   <body>
     <h1>Home page</h1>
     <p>TODO: replace with a proper template</p>
+    {{ csrfProtection(csrfToken) }}
```

Comment:
no form here, so a bit weird

Reply:
That's garbage, I just missed it. Thanks.